arrow-fx-test/src/main/kotlin/arrow/fx/test/eq/EqK.kt

```diff
@@ -63,3 +64,23 @@ fun Schedule.Decision.Companion.eqK(): EqK<DecisionPartialOf<Any?>> = object : E
         (if (l.cont) EQ.run { l.finish.value().eqv(r.finish.value()) } else true)
     }
 }
+
+fun <A> unsafeRunEq(fa: () -> A, fb: () -> A, EQA: Eq<A> = Eq.any()): Boolean {
+  val aa = try {
+    fa()
+  } catch (err: Throwable) {
+    err
+  }
+  val bb = try {
+    fb()
+  } catch (err: Throwable) {
+    err
+  }
+  return when {
+    aa is TimeoutException -> throw aa
+    bb is TimeoutException -> throw bb
+    aa is Exception && bb is Exception -> throwableEq().run { aa.eqv(bb) }
+    aa is Exception || bb is Exception -> false
```

Comment:
Should we re-throw here?

If we don't bubble up the exception, wouldn't we otherwise get an `expected true but found false` assertion here from `forAll`?

Reply:
In theory this should be equivalent to https://github.com/arrow-kt/arrow-fx/pull/228/files#diff-4399f003c25a9b249e3fab432f659bd4L191